src/Container.php

```diff
@@ -21,6 +21,11 @@ class Container implements ContainerInterface, ArrayAccess
     protected $items = [];
 
     /**
+     * @var array - \League\Container\ServiceProvider[]
```

Comment:
Should still just parse the arrayof class name as description shouldn't it? The type just being array. Actually never tested that, does the description HAVE to be above on properties?